css-positioning/absolute.css

```diff
@@ -0,0 +1,44 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url('images/one-does-not-simply.jpg');
+  background-size: cover;
+  position: relative;
+
+  /* your code here */
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+  position: absolute;
+  top: 10px;
+  left: 235px;
+
+  /* your code here */
+}
+
+.joke {
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  top: 158px;
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgba(0, 0, 0, 0.75));
+  position: absolute;
+  bottom: 50px;
+
+  /* your code here */
+}
```

Comment:
This isn't actually being applied properly. You can see that the image is still too bright. You need to utilize all of your offset properties, or give this a `width` / `height` so it can take up the full size of it's parent